src/handle-data.ts

```diff
@@ -75,7 +81,16 @@ export function getTotalPosition(layout) {
   return 'noTotals';
 }
 
-export default async function manageData(model, layout, pageInfo, setPageInfo) {
+export default async function manageData(
+  model: EngineAPI.IGenericObject,
+  layout: TableLayout,
+  pageInfo: PageInfo,
+  setPageInfo: stardust.SetStateFn<{
+    page: number;
+    rowsPerPage: number;
+    rowsPerPageOptions: number[];
+  }>
```

Comment:
doesn't the function take a `PageInfo` object as argument?

Reply:
Fixed